tests/serialization/test_object_template.py

```diff
@@ -23,7 +23,7 @@ def test_object_template_serde():
     assert copy_template == block_template
 
     # Tests below exercise similar code, but for measurement and process templates
-    pressure_template = ConditionTemplate("pressure", RealBounds(0.999, 1.001, 'atm'))
+    pressure_template = ConditionTemplate("pressure", RealBounds(0.1, 0.11, 'MPa'))
```

Comment:
Removal of `atm` is a little far, but it's easier to add than remove, so okay.

Reply:
Exactly.